ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4596,3 +4596,262 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - DTM
   opt_uses:
   - condensing_fan_current_sensor
+  
+AHU_CHWVM:
+  description: "AHU with chilled water valve monitoring and control"
+  is_canonical: true
+  implements:
+  - AHU
+  - CHWVM 
+  
+AHU_EPM:
+  description: "AHU with basic electrical parameter monitoring"
+  is_canonical: true
+  implements:
+  - AHU
+  - EPM
+
+AHU_FDPM:
+  description: "AHU with differential pressure monitoring across filter"
+  is_canonical: true
+  implements:
+  - AHU
+  - FDPM
+
+AHU_STC:
+  description: "AHU with supply air temperature monitoring and control"
+  is_canonical: true
+  implements:
+  - AHU
+  - STC   
+
+AHU_RTC:
+  description: "AHU with return air temperature monitoring and control"
+  is_canonical: true
+  implements:
+  - AHU
+  - RTC
+  
+AHU_ZA:
+  description: "AHU with zone air temperature and humidity sensor"
+  is_canonical: true
+  implements:
+  - AHU
+  - ZA
+
+AHU_VOADC2X:
+  description: "AHU with Variable outside air damper control and monitoring"
+  is_canonical: true
+  implements:
+  - AHU
+  - VOADC2X
+
+AHU_OAMC:
+  description: "AHU with outside sir control and monitoring"
+  is_canonical: true
+  implements:
+  - AHU
+  - OAMC 
+  
+AHU_DFMSC:
+  description: "AHU with discharge fan speed monitoring"
+  is_canonical: true
+  implements:
+  - AHU
+  - DFMSC
+
+AHU_RACO2C:
+  description: "AHU with return air CO2 monitoring and control"
+  is_canonical: true
+  implements:
+  - AHU
+  - RACO2C
+
+AHU_SFSS:
+  description: "AHU with supply fan run control and monitoring"
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS
+  
+AHU_ZA:
+  description: "AHU with zone air temperature monitoring and control"
+  is_canonical: true
+  implements:
+  - AHU
+  - ZA
+
+AHU_CO2C:
+  description: "AHU with zone air CO2 monitoring and control"
+  is_canonical: true
+  implements:
+  - AHU
+  - CO2C
+
+AHU_SFVSC:
+  description: "AHU with supply fan speed control"
+  is_canonical: true
+  implements:
+  - AHU
+  - SFVSC 
+
+AHU_RAIDS3X:
+  description: "AHU with return air isolation damper monitoring for three sets of dampers"
+  is_canonical: true
+  implements:
+  - AHU
+  - RAIDS3X
+
+AHU_RAIDS2X:
+  description: "AHU with return air isolation damper monitoring for two sets of dampers"
+  is_canonical: true
+  implements:
+  - AHU
+  - RAIDS2X
+
+AHU_RAIDS4X:
+  description: "AHU with return air isolation damper monitoring for four sets of dampers"
+  is_canonical: true
+  implements:
+  - AHU
+  - RAIDS4X
+
+AHU_SSPC:
```

Comment:
not valid: what kind of AHU is it? even if these are the only fields you get for the type, you still need to model it as the AHU type it really is (controls to zone temp, has dx or chw cooling, outside air monitoring, etc.). The translation will be adjusted to handle the missing fields. we do not want a bunch of bespoke AHUs which are really incomplete versions of a real, canonical AHU.

Reply:
Deleted the single Canonicals and have created unified canonials.